docs/source/intro/troubleshooting.rst

```diff
@@ -77,6 +77,64 @@ A way to do it is to add a line similar to the following to the ``~/.bashrc`` an
 
 .. _Stackoverflow link: http://stackoverflow.com/questions/21079820/how-to-find-pg-config-pathlink
 
+RabbitMQ Installation
+.....................
+
+If in ``verdi status`` RabbitMQ is not connected, first check that RabbitMQ is actually running:
+
+.. code-block:: console
+
+   $ rabbitmqctl status
+   Status of node rabbit@ph-tsm15-025 ...
+   [{pid,86960},
+   ...
+   {listeners,[{clustering,25672,"::"},{amqp,5672,"::"},{http,15672,"::"}]},
+
+By default, AiiDA profiles are configured to connect to RabbitMQ *via* ``amqp://guest:guest@127.0.0.1:5672``, hence this port should be open for connections.
+In Linux / Mac OSX you can also check which ports a PID has open using:
+
+.. code-block:: console
+
+   $ lsof -Pan -p 86960 -i
+   COMMAND    PID  USER   FD   TYPE             DEVICE SIZE/OFF NODE NAME
+   beam.smp 98979 user1   75u  IPv4 0x9d838dc03d5a2485      0t0  TCP *:25672 (LISTEN)
+   beam.smp 98979 user1   76u  IPv4 0x9d838dc047588625      0t0  TCP 127.0.0.1:58316->127.0.0.1:4369 (ESTABLISHED)
+   beam.smp 98979 user1   86u  IPv6 0x9d838dc034033ea5      0t0  TCP *:5672 (LISTEN)
+   beam.smp 98979 user1   87u  IPv4 0x9d838dc018071a15      0t0  TCP *:15672 (LISTEN)
+
+If a connection cannot be found, try starting ``rabbitmq-server`` in non-detached mode.
+If you encounter an output similar to that below, it may be that your versions of RabbitMQ and erlang (the programming language which RabbitMQ is written in) are incompatible.
```

Comment:
If this is the case, what user gonna to do to solve this? One additional description line would be better.

Reply:
Added in 210857b